backend/doc/apache.md

```diff
@@ -30,23 +30,36 @@ Create a file named `/etc/apache2/sites-available/tracim.conf` containing:
             Dav On
         </Directory>
         SetEnv proxy-sendcl
```

Comment:
@PhilippeAccorsi @grignards A comment here may be nice, did you know why do we need this
https://httpd.apache.org/docs/2.4/en/mod/mod_proxy_http.html said  that this can be inefficient:

>  However, it may require the entire request to be buffered by the proxy, so it becomes very inefficient for large requests.

Reply:
I thinks its for Webdav with macos or something like that. I can search in the history of this file to find commit.

Reply:
I opened an [issue](https://github.com/tracim/tracim/issues/3320) for this subject, we'll keep this directive until it is addressed.